### PR TITLE
Adding ability to pass `clip_interval` and `vmin/vmax` to MapSequence.plot()

### PR DESCRIPTION
**Pull request by hayesla**
Addresses #7252

### CODE REVIEW

**Comment by wtbarnes** [comment]
This is great, though the amount of code we need to repeat both here and in `GenericMap.plot` is worrying. I don't know of a good way to avoid that though without an overhaul of how we handle visualization.

This should probably get a figure test as well.

**Comment by nabobalis** [comment]
> This is great, though the amount of code we need to repeat both here and in `GenericMap.plot` is worrying. I don't know of a good way to avoid that though without an overhaul of how we handle visualization.
> 
> This should probably get a figure test as well.

Have my terrible idea.

File: sunpy/map/tests/test_mapsequence.py [hunk 1]
```diff
@@ -230,3 +230,10 @@ def test_save(aia171_test_map, hmi_test_map, tmp_path):
     for k in seq.maps[1].meta:
         assert test_seq.maps[1].meta[k] == seq.maps[1].meta[k]
     assert_quantity_allclose(test_seq.maps[1].data, seq.maps[1].data)
+
+
+@figure_test
+def test_map_sequence_plot_clip_interval(aia171_test_map):
+    seq = sunpy.map.Map([aia171_test_map, aia171_test_map, aia171_test_map], sequence=True)
+    animation = seq.plot(cmap='Greys', clip_interval=(1,99.5)*u.percent)
+    animation._step()
```
Comment (nabobalis) [inline sunpy/map/tests/test_mapsequence.py:239]: Need to check this works and add the hash

**Review by nabobalis — APPROVED** [review]

**Review by wtbarnes — APPROVED** [review]
We should probably get another review from someone who didn't write most of the code.

**Pull request merged**